presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -982,7 +1009,7 @@ public ConnectorTableHandle createTemporaryTable(ConnectorSession session, List<
                         .map(handle -> new Column(handle.getName(), handle.getHiveType(), handle.getComment()))
                         .collect(toImmutableList()))
                 .withStorage(storage -> storage
-                        .setStorageFormat(fromHiveStorageFormat(storageFormat))
+                        .setStorageFormat(fromHiveStorageFormat(finalStorageFormat))
```

Comment:
@wenleix Very good point, now it is possible to have a mixed types of formats. I included format name to table name, like `__presto_temporary_table_ORC...`  or `__presto_temporary_table_PAGEFILE...` . Do you have other idea?

Reply:
@viczhang861 : Sounds reasonable. I am always thinking to refactor the temporary table name to have some hint over which stage it comes from (instead of an UUID). But let's keep it as is 😃